initialize_vault_store.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+from authx.auth import set_service_store_secret, add_provider_to_opa
+import sys
+
```

Comment:
Might want to add a comment & #! at the top letting people know what this script is doing

Reply:
done